host-controller/src/main/java/org/jboss/as/domain/controller/operations/HostProcessReloadHandler.java

```diff
@@ -76,10 +76,13 @@
             .setAlternatives(ModelDescriptionConstants.USE_CURRENT_DOMAIN_CONFIG)
             .build();
 
+    protected static final AttributeDefinition TIMEOUT = new SimpleAttributeDefinitionBuilder(ModelDescriptionConstants.TIMEOUT, ModelType.INT, true)
+            .setDefaultValue(new ModelNode(0)).build();
```

Comment:
> Does the suspend operation block?

Thanks for the tip!, yes, although the HC reload operation response is received immediately, the HC is restarted when all managed servers have been stopped, with the changes in this issue, the suspend operation blocks and later stops the managed servers. So, yes, you're right, independently of the previous suspend configurations done for managed servers, reload will always wait until all servers are stopped. That means (I think) it makes no sense configure different suspends for managed servers if you are going to use a reload HC. If that reload without timeout should overwrite any previous suspend with 0 timeout, is still a minor doubt for me.

Additionally, suspend operation used in this PR should be using BlockingQueueOperationListener as you did in #1490 ,that something still pending to do here, I will change it.

I have other doubts about if suspend operation is working fine anyway, I will write those thoughts in the Jira issue instead here.